Samples/AudioCreation/cs/AudioCreation/Scenario7_RecordPlayback.xaml.cs

```diff
@@ -0,0 +1,589 @@
+//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+
+using SDKTemplate;
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Windows.Devices.Enumeration;
+using Windows.Foundation;
+using Windows.Media;
+using Windows.Media.Audio;
+using Windows.Media.Capture;
+using Windows.Media.Devices;
+using Windows.Media.Render;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+
+// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace AudioCreation
+{
+    /// <summary>
+    /// This scenario shows using AudioGraph for audio capture from a microphone with low latency
+    /// to a memory buffer, and looping playback of the captured audio buffer.
+    /// </summary>
+    public sealed partial class Scenario7_RecordPlayback : Page
+    {
+        MainPage rootPage;
+        AudioGraph graph;
+        AudioDeviceOutputNode deviceOutputNode;
+        AudioDeviceInputNode deviceInputNode;
+        AudioFrameOutputNode frameOutputNode;
+        AudioFrameInputNode frameInputNode;
+
+        // These variables are synchronization points; they experience the following
+        // state transitions:
+        // requestStartRecording: false -> true (UI thread, user clicks Start Recording)
+        // requestStartRecording: true -> false (audio thread, when it registers request)
+        // isRecording: false -> true (audio thread, when it registers request)
+        // isRecording: true -> false (UI thread, user clicks Stop Recording)
+        // isRecording: true -> false (audio thread, buffer reaches maximum capacity)
+        //
+        // And similarly for requestStartPlaying and isPlaying.
+        //
+        volatile bool requestStartRecording;
+        volatile bool requestStartPlaying;
+        volatile bool isRecording;
+        volatile bool isPlaying;
+        volatile int audioFrameCount;
+
+        DateTime recordingStartTime;
+        int lastStatusMsec;
```

Comment:
I get a compiler warning saying that this variable is never used. Please remove it.